bugwarrior/services/deck.py

```diff
@@ -0,0 +1,184 @@
+import datetime
+import logging
+import pathlib
+import base64
+
+import pydantic
+from typing import List
+import requests
+import typing_extensions
+from dateutil.tz import tzutc
+
+from bugwarrior import config
+from bugwarrior.services import IssueService, Issue, ServiceClient
+
+log = logging.getLogger(__name__)
+
+
+class NextcloudDeckConfig(config.ServiceConfig, prefix='deck'):
+    service: typing_extensions.Literal['deck']
+    base_uri: pydantic.AnyUrl
```

Comment:
I suspect we should use `config.StrippedTrailingSlashUrl` here so if somebody adds an extra slash at the end in their config it doesn't break url construction.